modules/nodelist.php

```diff
@@ -58,7 +58,14 @@
 	$ng = $_GET['ng'];
 $SESSION->save('nlng', $ng);
 
-$nodelist = $LMS->GetNodeList($o, NULL, NULL, $n, $s, $g, $ng);
+$page = (!isset($_GET['page']) ? 1 : intval($_GET['page']));
+$per_page = intval(ConfigHelper::getConfig('phpui.nodelist_pagelimit', $listdata['total']));
+$offset = ($page - 1) * $per_page;
+$total = intval($LMS->GetNodeList($o, NULL, NULL, $n, $s, $g, $ng, null, null, true));
+
+$nodelist = $LMS->GetNodeList($o, NULL, NULL, $n, $s, $g, $ng, $per_page, $offset);
```

Comment:
To trzeba byłoby do sterowników mysql, mysqli i postgres w LMS dodać odpowiednią metodę, która zasłoniłaby przez twórcą zapytań SQL w LMS specyfikę bazy danych. Ktoś chętny to drobienia tego? ;-)